### PR TITLE
assets-client-sdk conversion

### PR DESCRIPTION
**Pull request by nkrijnen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,8 @@
-<html>
 
+<html>
 <head>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.class.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.cookie.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/elvis_api/js/jquery.elvis.js" type="text/javascript"></script>
+    <script src="assets-client-sdk.min.js"></script>
+    <script src="jquery.js" type="text/javascript"></script>
```
Comment (jaapvanblaaderen) [inline index.html]: Maybe include jquery from a CDN?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,8 @@
-<html>
 
+<html>
 <head>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.class.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/jquery.cookie.js" type="text/javascript"></script>
-	<script src="${pluginsBaseRootUrl}/web.shared/elvis_api/js/jquery.elvis.js" type="text/javascript"></script>
+    <script src="assets-client-sdk.min.js"></script>
```
Comment (jaapvanblaaderen) [inline index.html]: Should this include a version number? Now it's not really clear if this is outdated or not. Maybe we should publish it to a CDN like cdnjs?

**Review by jaapvanblaaderen — COMMENTED** [review]
Nice to see some updates to the good old stamper :-) Left a few comments.

File: index.html (hunk 2 above)
Comment (nkrijnen) [reply index.html]: Good ideas 👍

File: index.html (hunk 1 above)
Comment (jaapvanblaaderen) [reply index.html]: Also maybe try if it works with the latest (and secure) jQuery?

Comment (nkrijnen) [reply index.html]: Used unpkg.com CDN and tested with latest version of jQuery, seems to be working fine - the plugin uses mostly basic jQuery constructs.

File: index.html (hunk 2 above)
Comment (nkrijnen) [reply index.html]: Used unpkg.com CDN to auto-load latest client sdk directly from the published NPM package. Tested with latest version of the sdk, works fine.

**Review by kaas93 — APPROVED** [review]

**Review by jeroendekker — APPROVED** [review]

**Pull request merged**